stdlib/http/src/main/java/org/ballerinalang/net/http/DataContext.java

```diff
@@ -38,21 +38,21 @@
     private Strand strand;
     private HttpClientConnector clientConnector;
     private BObject requestObj;
-    private NonBlockingCallback callback;
+    private BalFuture future;
     private HttpCarbonMessage correlatedMessage;
 
-    public DataContext(Strand strand, HttpClientConnector clientConnector, NonBlockingCallback callback,
+    public DataContext(Strand strand, HttpClientConnector clientConnector, BalFuture future,
```

Comment:
Do we still need Strand?

Reply:
It's still used for `dataContext.getStrand().getProperty(HttpConstants.REMOTE_ADDRESS)` so can't remove if fully yet.